src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,51 @@
+.nav {
+
+  &__items {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+  }
+
+  &__item {
+    display: flex;
+
+    &:not(:first-child) {
+      padding-left: 20px;
+    }
+
+    &:first-child {
+      padding-left: 0;
+    }
+  }
+
+  &__link {
+    padding: 23px 0;
+    line-height: 14px;
+    font-size: 12px;
+    font-weight: 500;
+    color: $color-main;
+    text-transform: uppercase;
+    transition: color 0.5s;
+
+    &:hover {
+      color: $color-links;
+    }
+  }
+}
+
+.is-active {
```

Comment:
let's create separate active modifier for the links, and move these styles to those modifier
and will use is-active class only in html for tests passing